.github/workflows/chatops.yml

```
@@ -448,8 +448,8 @@ jobs:
             git config --global user.name "vdaas-ci"
             git config --global user.email "ci@vdaas.org"
 
-            git add .
-            git commit --signoff -m ":robot: Update license headers and formatting go codes"
+            git add cmd example hack internal k8s pkg
```

how about this?
```suggestion
            git add .
            git rm go.mod go.sum
```

---

Thanks.
`git rm go.mod go.sum` removes **go.mod and go.sum file**.
So it should be
```suggestion
            git add .
            git reset go.mod go.sum
```

---

how about it?

---

right... I think you are right :)
LGTM